steps/workforce/allocations.ts

```diff
@@ -34,3 +35,11 @@ export const allocateCase = async (page: Page, crn: string, practitioner: Practi
     await page.locator('button', { hasText: 'Allocate Case' }).click()
     await page.locator('div.govuk-panel--confirmation >> h1.govuk-panel__title', { hasText: 'Allocation complete' })
 }
+
+const selectTeam = async (page: Page) => {
+    const selectRequired = (await page.locator('h1', { hasText: 'Select your teams' }).count()) > 0
+    if (selectRequired) {
+        await page.locator('label', { hasText: ' Wrexham - Team 1 ' }).click()
```

Comment:
You should be able to get this value from `data.teams.allocationsTestTeam.teamName`